samples/http-cloudevents/server.py

```diff
@@ -23,6 +23,18 @@
 def home():
     # create a CloudEvent
     event = from_http(request.get_data(), request.headers)
+    print(event)
+
+    # you can access cloudevent fields as seen below
+    print(
+        f"Found CloudEvent from {event['source']} with specversion {event['specversion']}"
+    )
+
+    if event["type"] == "com.example.sampletype1":
+        print(f"CloudEvent {event['id']} is binary")
+
+    elif event["type"] == "com.example.sampletype2":
+        print(f"CloudEvent {event['id']} is structured")
```

Comment:
Not sure if the event type should determine if it's binary or structured.

I'd probably just remove this as we already print the whole event.

Reply:
Truth be told, I only added that if statement as a means of showing the user how to check the event's type and id given we don't have getters/setters on that in CloudEvent yet. I would still like to show case member access on those required fields so suggestions on that is welcome.